graylog2-server/src/main/java/org/graylog2/dashboards/widgets/DashboardWidgetCreator.java

```diff
@@ -20,15 +20,13 @@
 import com.codahale.metrics.MetricRegistry;
 import com.mongodb.BasicDBObject;
 import org.graylog2.indexer.searches.Searches;
-import org.graylog2.indexer.searches.timeranges.AbsoluteRange;
-import org.graylog2.indexer.searches.timeranges.InvalidRangeParametersException;
-import org.graylog2.indexer.searches.timeranges.KeywordRange;
-import org.graylog2.indexer.searches.timeranges.RelativeRange;
-import org.graylog2.indexer.searches.timeranges.TimeRange;
+import org.graylog2.indexer.searches.timeranges.*;
```

Comment:
Please don't use wildcard imports.

Instructions for IntelliJ IDEA: http://myshittycode.com/2013/12/30/intellij-preventing-wildcard-imports/